librz/core/cmd/cmd_api.c

```diff
@@ -1110,15 +1115,19 @@ static void do_print_child_help(RzCmd *cmd, RzStrBuf *sb, const RzCmdDesc *cd, c
 	rz_strbuf_appendf(sb, "%s\n", pal_reset);
 }
 
-static void print_child_help(RzCmd *cmd, RzStrBuf *sb, RzCmdDesc *cd, size_t max_len, bool use_color) {
-	do_print_child_help(cmd, sb, cd, cd->name, cd->help->summary ? cd->help->summary : "", true, max_len, use_color);
+static void print_child_help(RzCmd *cmd, RzStrBuf *sb, RzCmdDesc *cd, size_t max_len, const char *vertical_line, bool use_color) {
+	do_print_child_help(cmd, sb, cd, cd->name, cd->help->summary ? cd->help->summary : "", vertical_line, true, max_len, use_color);
 }
 
 static char *group_get_help(RzCmd *cmd, RzCmdDesc *cd, bool use_color) {
+	bool scr_utf8 = cmd->core ? cmd->core->config ? rz_config_get_b(cmd->core->config, "scr.utf8") : false : false;
+	bool scr_curvy = scr_utf8 && rz_config_get_b(cmd->core->config, "scr.utf8.curvy");
```

Comment:
if `cmd->core` or `cmd->core->config` is NULL, then this will crash at line 1124.

just set them to `false` by default, then have an `if (cmd->core && cmd->core->config) {` and set them.

Reply:
the issue is that you are testing only the first line. i strongly suggest to just define an inline function which tests for `cmd->core` and `cmd->core->config` to be non-null and return a boolean for any given parameter of `rz_config_get_b`, so you can re-use it anywhere and is safe.

Also there is no need to do `a ? b ? c ...` just test them via `a && b && c ...` since if a is false, the whole thing will just fail before testing b and c.